src/test/integration/completion.test.ts

```diff
@@ -13,6 +13,7 @@ suite('completion', () => {
       new vscode.CompletionItem('data', vscode.CompletionItemKind.Class),
       new vscode.CompletionItem('locals', vscode.CompletionItemKind.Class),
       new vscode.CompletionItem('module', vscode.CompletionItemKind.Class),
+      new vscode.CompletionItem('moved', vscode.CompletionItemKind.Class),
```

Comment:
This was required several months ago, because the version of terraform in CI did not have the `moved' block we added as completable. I'll check if this is still the case now several months later.

Reply:
I see - that makes sense!

Reply:
So, if we leave terraform version in CI as `~1.0` and remove the `moved` completion here, everything passes in CI.

However, locally, if you have 1.1.X or greater, the tests will fail:

```

  code actions
    ✔ supported actions (3233ms)
  completion
    1) simple completion
  document symbols
    ✔ returns symbols (92ms)
  2 passing (3s)
  1 failing
  1) completion
       simple completion:

      AssertionError [ERR_ASSERTION]: Expected values to be strictly deep-equal:
+ actual - expected

+ 'moved'
- 'output'
      + expected - actual

      -moved
      +output
```

Reply:
Originally I wasn't commenting so much on what the constraint should be, but more asking what the syntax means. Admittedly it's a question I can and should answer with docs search myself instead of lazily asking you 😅 

https://github.com/hashicorp/setup-terraform#inputs
https://www.npmjs.com/package/semver#user-content-tilde-ranges-123-12-1

TL;DR I'm assuming `~1.0` is actually equivalent to `~1`,  `1.x` or `>= 1.0.0 < 2.0.0`. Given that it's equivalent I don't understand the reason for changing `~1.0` to `~1` but I expect it's a harmless change, although the failing tests would suggest otherwise?

To answer your question of what version should we be testing against - I'd vote for either any latest _compatible_ version (i.e. changing to `~1.1` when `moved` block was introduced) or latest major version (i.e. keeping it as is). Compatible ~= meaning that all tests keep passing if we run them against any version which satisfies the constraint. 

The downside of latest major version is that our tests are little more fragile, but if we claim compatibility with any `v1.x` version to our users, then maybe this is "by design".